CoffeeChat/SceneDelegate.swift

```diff
@@ -27,11 +27,10 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         if let signIn = GIDSignIn.sharedInstance(), signIn.hasPreviousSignIn() {
             signIn.restorePreviousSignIn()
             // Onboard user if they haven't done so yet, otherwise bring to home.
-            let onboardingCompleted = UserDefaults.standard.bool(forKey: Constants.UserDefaults.onboardingCompletion)
+//            let onboardingCompleted = UserDefaults.standard.bool(forKey: Constants.UserDefaults.onboardingCompletion)
+            let onboardingCompleted = false
```

Comment:
are we setting this to false to force onboarding to show for demo purposes, or is this leftover from testing?